hyrax/test/fixtures/jp2_fits.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why are these files stored in the ``test`` directory, not ``spec``?

Reply:
Good point, I've moved those and removed the unused `test/` stubs